fearless/Common/Network/Subquery/SubqueryHistoryElementData.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+import FearlessUtils
+
+struct SubqueryHistoryElementData {
```

Comment:
Maybe rename this to `SubqueryHistoryRewardData` because we might have another history elements